CLI/test/CLITest.cs

```diff
@@ -189,6 +189,16 @@ public void TestReplacingSyntaxOption()
             CLITestHelper.Test("return_code_4", ReturnCode.Warning);//1
         }
 
+        /// <summary>
+        /// Try parsing with copy is inside a typedef.
```

Comment:
'when copy is inside a typedef' ? or 'with copy inside a typedef'

Reply:
modified: with copy inside a typedef.
https://github.com/TypeCobolTeam/TypeCobol/pull/1648/commits/31338d478e73bd6e55f1aca16dd21ebabd1e9eff